test/regress/tests/router/expected/mixed_routing.out

```diff
@@ -204,12 +204,12 @@ NOTICE: send query to shard(s) : sh1
 ----
 (0 rows)
 
-SELECT * FROM xxmixeduuid WHERE id <= 'FFFFFFFF-FFFF-FFFF-FFFF-FFFFFFFFFFFF' ORDER BY id;
+SELECT * FROM xxmixeduuid WHERE id <= 'FFFFFFFF-FFFF-FFFF-FFFF-FFFFFFFFFFFE' ORDER BY id;
```

Comment:
Maybe rollback all changes in test/regress/tests/router (except this file)? Because this changes seem unneeded

Reply:
And router/qrouter/proxy_routing_test.go

Reply:
Well I didn't really check what tests need correction, I just wanted to keep test logic unchanged